gc/base/GCExtensionsBase.hpp

```diff
@@ -1796,6 +1798,8 @@ class MM_GCExtensionsBase : public MM_BaseVirtual {
 #if defined(OMR_VALGRIND_MEMCHECK)
 		, valgrindMempoolAddr(0)
 		, memcheckHashTable(NULL)
+		, portableSharedCacheEnabled(false)
+		, portableSharedCacheExplicitlySet(true)
```

Comment:
Considering name this default does not make sense (why we set Portable Shared Cache Explicitly set?!!)
I believe name should be changed to something like `forceCeilingShiltForContainers`